usagov/usagov-2021
# USAGOV-1896-error-gap:

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1896

## Description

Remove padding below checkboxes to align red error line to bottom of the input.

Use javascript to inspect position of each checkbox to mark the ones that are the end of a column in the multi-column layout so that we can remove the padding below just those element.

## Type of Changes


![image](https://github.com/user-attachments/assets/7cad31ef-0022-4fb3-a819-c72dd3ea8681)

![image](https://github.com/user-attachments/assets/b65cdbf0-bab0-4b01-86ac-2a999dc1ab03)


## Testing Instructions



### Change Requirements



### Validation Steps

- On mobile, go to benefits search page (new benefit finder landing page) and don't select any categories. Press "Apply Selections" confirm the vertical red border does not extend below the checkbox label.
- On desktop, go to benefits search page (new benefit finder landing page) and don't select any categories. Press "Apply Selections" confirm the vertical red border does not extend below the checkbox label.
- On desktop, go to benefits search page (new benefit finder landing page) and don't select any categories. Press "Apply Selections" confirm the vertical red border does not extend below the checkbox label. Resize window width and confirm gaps don't appear between checkboxes as text wraps and options shift columns

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
